cli/types/index.d.ts

```diff
@@ -1,4856 +1,4856 @@
-// Project: https://www.cypress.io

```

Comment:
@mboudreau it looks like your linter rewrote this entire file making a huge diff, can you please revert the changes that are not relevant to this PR?

Reply:
I dunno what's happening here, I didn't reformat the file.  When I do a diff between my local repo changes and cypress develop, it only changes it highlights are the optionals I added.  I'll see if I can't just revert it back to develop and re-add my changes.

Reply:
Okay, reverted it and seems to have been fixed.